### PR TITLE
Feature/nav bar

### PR DESCRIPTION
**Pull request by calhong**
image

love me some multi-sized icons because I don't have photoshop

### CODE REVIEW

**Review by ianmah — CHANGES_REQUESTED** [review]
use yarn hEH delete package-lock.json

**Review by ianmah — APPROVED** [review]
We could use CapitalizedCamelCase for the icons but doesnt rly matter `HomeIcon`

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import './App.css';
 import Button from './components/Button'
 import SongList from './components/SongList';
+import NavBar from "./components/NavBar";
```
Comment (ianmah) [inline src/App.js:4]:
```suggestion
import NavBar from './components/NavBar';
```
ahaaaaaa

**Comment by calhong** [comment]
> We could use CapitalizedCamelCase for the icons but doesnt rly matter `HomeIcon`

will update on icon change to correct sizing and blank backgrounds

**Pull request merged**